### PR TITLE
feat(UI-1042): add missing ui fields in webhook trigger form

### PR DESCRIPTION
**Pull request by RonenMars**
## Description
When a user creates/edits a webhook trigger, they should also have these 2 fields:

HTTP Method (selection between: GET, HEAD, POST, PUT, DELETE, OPTIONS, PATCH - upper case in the UI, but lower case in the trigger definition) - see https://docs.autokitteh.com/integrations/http/events/#event-types and https://docs.autokitteh.com/integrations/http/events#option-1-single-event-type

CEL Filter Expression - same as "filter" in other integration-based triggers - see https://docs.autokitteh.com/integrations/http/events#option-2-cel-filter-expression

Both are optional. Also, mention the option-1 and option-2 URLs as help links for users.

## Linear Ticket
https://linear.app/autokitteh/issue/UI-1042/missing-ui-fields-in-webhook-trigger

## What type of PR is this? (check all applicable)

-   [x] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/components/organisms/triggers/formParts/webhook.tsx [hunk 1]
```diff
@@ -1,38 +1,136 @@
...
+					<ExternalLinkIcon className="size-3.5 fill-green-800 duration-200" />
+				</Link>
+				<Link
+					className="group inline-flex items-center gap-2.5 pl-1 text-green-800"
+					target="_blank"
+					to={t("webhookInfo.secondLink")}
+				>
+					{t("webhookInfo.secondLinkText")}
+					<ExternalLinkIcon className="size-3.5 fill-green-800 duration-200" />
+				</Link>
```
Comment (J1za) [inline src/components/organisms/triggers/formParts/webhook.tsx:100]: maybe it's better to add everything to the accordion component?

File: src/components/organisms/triggers/formParts/webhook.tsx [hunk 2]
```diff
@@ -1,38 +1,136 @@
 import React, { useEffect, useState } from "react";
 
+import { Controller, useFormContext, useWatch } from "react-hook-form";
 import { useTranslation } from "react-i18next";
 
 import { cn, getApiBaseUrl } from "@src/utilities";
+import { TriggerFormData } from "@validations/trigger.schema";
```
[the remark this answers is not in the document]
Comment (RonenMars) [reply src/components/organisms/triggers/formParts/webhook.tsx:7]: fixed

File: src/locales/en/tabs/translation.json [hunk 3]
```diff
@@ -129,7 +138,8 @@
 				"selectTriggerType": "Select trigger type",
 				"triggerType": "Trigger type",
 				"value": "Value",
-				"eventTypesSelect": "Enter the event type"
+				"eventTypesSelect": "Enter the event type",
```
[the remark this answers is not in the document]
Comment (RonenMars) [reply src/locales/en/tabs/translation.json:141]: fixed

File: src/components/organisms/triggers/formParts/webhook.tsx (hunk 1 above)
Comment (RonenMars) [reply src/components/organisms/triggers/formParts/webhook.tsx:100]: Refactored:
image

**Review by J1za — APPROVED** [review]

**Pull request merged**